README.md

```diff
@@ -12,4 +12,7 @@ Remove Viewed Products for Magento 2 removes the recently viewed products featur
 There is no configuration.
 
 ## Copyright
-&copy; 2020 CustomGento GmbH
+© 2020 CustomGento GmbH - Present
+
+## License
```

Comment:
Please put this before the copyright. Let us always use the order from our [official docs](https://docs.customgento.com/tier-prices-for-configurable-products-m2.html).